client/src/components/RecommendView/RecommendView.jsx

```diff
@@ -85,10 +85,12 @@ function RecommendView() {
 
   return (
     <div className="recommend-view">
+      <NavBar />
       <div className="users">
-        <h2>Users</h2>
-        <input type="text" placeholder='Search For A User' value={userSearch} onChange={(e) => setUserSearch(e.target.value)}/>
-        {displayUsers ? displayUsers.map((item, idx) => {
+        <div className="header">
+          <input type="text" placeholder='Search For A User' value={userSearch} onChange={(e) => setUserSearch(e.target.value)}/>
+        </div>
+        {displayUsers?.length !== 0 && displayUsers !== null ? displayUsers?.map((item, idx) => {
```

Comment:
I left this ternary expression here because I think it looked weird to have a big chunk of html outside the return.

Reply:
How can the length not be 0 but the object be null? 🤔

Reply:
It's checking that the length is not 0 and that the object is not null.